modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,108 @@ It also includes enhancements to the Eventing and Search services.
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-605]
+== Release 6.0.5
+
+Couchbase Server 6.0.5, released in January 2021, is the fifth maintenance release in the 6.0.x series for Couchbase Server. 
+This release includes bug fixes related to many components.
+
+[#fixed-issues-605]
+=== Fixed Issues
+
+The notable fixes in this release are listed below.
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v605-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-43416[MB-43416^]
+| *Summary*: In a high bucket density test (with 30 buckets) with replication on, goxdcr process on source cluster data node was observed to exit multiple times with `status 2` and a runtime error.
+
+| https://issues.couchbase.com/browse/MB-40846[MB-40846^]
+| *Summary*: Fixed an incorrect XDCR stream request rollback caused by a consumer ahead of producer error.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v605-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-41804[MB-41804^]
+| *Summary*: Fixed an issue where ephemeral buckets set to `auto_delete` with replicas configured could reach a deadlock due to eviction.
+
+| https://issues.couchbase.com/browse/MB-41255[MB-41255^]
+| *Summary*: Fixed an issue where the replication of deletes with XATTRS failed if a EWOULD_BLOCK occurred.
+
+| https://issues.couchbase.com/browse/MB-40531[MB-40531^]
+| *Summary*: Fixed an issue where the HiFi-MRU Item Pager evicted items from active vBuckets even when there were ample replica vBucket items which could be evicted first.
+
+| https://issues.couchbase.com/browse/MB-38741[MB-38741^]
+| *Summary*: The frequency at which the expiry pager runs can be set using a configuration parameter `exp_pager_stime`. Since the default value of 3600s was seen to impact front-end operations, the default value has been reduced to 600s.
+
+| https://issues.couchbase.com/browse/MB-38283[MB-38283^]
+| *Summary*: Fixed an issue where Memcached log wrapped due to excessive DCP "no memory" messages.
+
+| https://issues.couchbase.com/browse/MB-34949[MB-34949^]
+| *Summary*:  Fixed an issue where StateLock was not held in all cases.
```

Comment:
Can probably expand this a bit - something like:

Fixed a theoretical race condition where if a vBucket changed state away from active at the same time as a Delete operation, the delete could incorrectly report success; but would not be applied to new active vBucket.